src/panels.cpp

```diff
@@ -2061,6 +2061,56 @@ static void draw_mana_wide( const player &u, const catacurses::window &w )
     print_mana( u, w, " %s: %s %s : %s", -5, -5, 13, -5 );
 }
 
+static void print_weary( const player &u, const catacurses::window &w, const std::string &fmt_string,
+                         const int j1, const int j2, const int j3 )
+{
+    werase( w );
+
+    auto weary_pair = get_hp_bar( u.weary_threshold(), u.weariness() );
+    const int exertion_range = EXTRA_EXERCISE;
+    const int exertion = EXTRA_EXERCISE - u.attempted_activity_level;
+    auto exertion_pair = get_hp_bar( exertion_range, exertion );
```

Comment:
1. Less auto, please.
2. These HP bars are very unclear. The exertion bar appears to do nothing, whether I'm digging a pit, or just sitting there waiting. A HP-bar style display would work for how far you are to the next level of weariness, but that's going to be fairly closely tied to how weariness is calculated, not how this is currently implemented. Exertion and weariness should be state displays, like hunger, or tiredness, or temperature.

Reply:
1. Sure, though auto is used almost everywhere else for `get_hp_bar`, probably because `std::pair<std::string, nc_color>` is a mess.
2. As mentioned in the original post, the exertion bar does nothing.  It's intended to reflect how badly your most recent activity is draining your weariness, but I'll just remove it.